### PR TITLE
Update prompt types documentation with new date prompt types

### PR DESCRIPTION
**Pull request by EstherPhebe**
closes odk-x/tool-suite-X#289

#### What is included in this PR?
Update prompt types documentation to include
1. Year only prompts
2. Month only prompts
3. Year and month only
4. Non-gregorian calendar date pickers.

#### What new issues will need to be opened because of this PR?
None
#### What is left to be done in the addressed issue?
None
#### What problems did you encounter?
None

### CODE REVIEW

**Comment by EstherPhebe** [comment]
https://app.circleci.com/pipelines/github/odk-x/docs/388/workflows/0e2dceaa-0ab5-4ad4-ac5e-518718cdbf0b/jobs/807

I think something is wrong with the build pipeline.

**Comment by elmps2018** [comment]
Looks like one of the checks failed on circleci

File: odkx-src/xlsx-converter-reference.rst [hunk 1]
```diff
@@ -329,12 +337,42 @@ The following prompt types are available in ODK-X Survey.
...
+    - | Uses a date picker widget to capture Persian dates (Non-Gregorian Calendar)
+  * - taiwan_calendar_picker 
+    - | Uses a date picker widget to capture Taiwan dates (Non-Gregorian Calendar)
+  * - thai_calendar_picker
+    - | Uses a date picker widget to capture Thai dates (Non-Gregorian Calendar)
+  * - ummalqura_calendar_picker
+    - | Uses a date picker widget to capture Umm al-Qura dates (Non-Gregorian Calendar)
   * - date
     - | Uses a date picker widget to capture a date. Automatically adjusts for timezone.
+  * - date (non-gregorian)
```
Comment (elmps2018) [inline odkx-src/xlsx-converter-reference.rst:364]: How is this different from the specific non-Gregorian calendars listed above? Not clear to me.

File: odkx-src/xlsx-converter-reference.rst [hunk 2]
```diff
@@ -395,8 +433,11 @@ The following prompt types are available in ODK-X Survey.
   * - video
     - | Used to capture a video.
   * - textarea
-    - | Used to enter the information in a big text area or paragraphs.
-    
+    - | Used to enter the information in a big text area or paragraphs. 
+
+.. note::
+  The Non-Gregorian dates are saved to the database in a converted Gregorian date time but is displayed to the user as a Gregorian date.
```
Comment (elmps2018) [inline odkx-src/xlsx-converter-reference.rst:439]: Is it displayed as Gregorian or non-Gregorian?

**Review by elmps2018 — COMMENTED** [review]
Great to see this getting documented! Had a few comments. Expect this is another one to finalize but not merge until apps are updated.

File: odkx-src/xlsx-converter-reference.rst (hunk 2 above)
Comment (EstherPhebe) [reply odkx-src/xlsx-converter-reference.rst:439]: Non-gregorian, actually. I'll correct it.

**Comment by EstherPhebe** [comment]
image

This is why the checks are failing and I'm not sure how to fix it.

File: odkx-src/xlsx-converter-reference.rst (hunk 2 above)
Comment (elmps2018) [reply odkx-src/xlsx-converter-reference.rst]: Great, thanks for fixing.

**Review by elmps2018 — COMMENTED** [review]
This looks like it is now lower-case, is that correct?

**Comment by wbrunette** [comment]
added milestone for release 2.1.9. This PR should not be merged until the release.

**Pull request merged**